src/components/toast/toast.js

```diff
@@ -289,18 +289,16 @@ function MdToastProvider($$interimElementProvider) {
           // Root element of template will be <md-toast>. We need to wrap all of its content inside of
           // of <div class="md-toast-content">. All templates provided here should be static, developer-controlled
           // content (meaning we're not attempting to guard against XSS).
-          var parsedTemplate = angular.element(template);
-          var wrappedContent = '<div class="md-toast-content">' + parsedTemplate.html() + '</div>';
+          var parsedTemplate = angular.element(template.replace(/<!--(.*?)-->/g, ''));
```

Comment:
This doesn't work because Angular actually lets you apply directives as comments.

https://docs.angularjs.org/guide/directive